lte/protos/sctpd.proto

```diff
@@ -25,7 +25,9 @@ message InitReq {
     repeated string ipv6_addrs = 4; // ipv6 addresses to listen on
     uint32 port = 5; // port to listen on
     uint32 ppid = 6; // ppid used with new associations
-    bool force_restart = 7; // whether to force a new sctp connection setup
+    uint32 ngap_port = 7; // ngap port to listen on
+    uint32 ngap_ppid = 8; // ngap ppid used with new associations
```

Comment:
why do you define separate fields for these? you should use port and ppid fields by passing the NGAP port and ppid.